### PR TITLE
Added support for Sun Non-US Backslash key

### PR DESCRIPTION
**Pull request by rxrbln**
I soldered and flashed a keyboard converter for Sun serial keyboards last night and it worked great except the ISO "<>|" key. After tinkering a little bit on the mapping to find out what keycode is generated, this change makes the key work on my German ISO keyboard. The keyboards defines are a bit convoluted and hard to follow, so I'm not 100% sure if this also should be added additionally in any of the other tables and defines in the file, ..?

### CODE REVIEW

**Comment by knightjp** [comment]
Does the left cluster work? I have the Type 7 with the standard USB connectors. The keyboard works great, but I want the functionality of the left cluster. It seems that there seems to be no way to get it to work on Windows. I'm not a programmer, so I don't know. I've been searching for some kind of Windows driver to give me that functionality.

**Comment by tmk** [comment]
@knightjp SUN converter doesn't support USB keyboard, only for Type-5, 4 and 3.
I think you can remap the keys with my USB-USB converter but it requires hardware.
https://github.com/tmk/tmk_keyboard/tree/master/converter/usb_usb
I don't have knowledge about windows driver at all.


@rxrbln I believe the ISO key is supported now if you are still interested. Sorry for late reply.

https://github.com/tmk/tmk_keyboard/blob/8898747b098b02ad60b278da6abbb4bae6029033/converter/sun_usb/unimap_trans.h#L31-L41

Or you can try this converter project also.
https://github.com/xelalexv/suniversal

**Review by Janeythor — APPROVED** [review]

**Comment by tmk** [comment]
Fixed `keymap.c`  with c10c892.

But keymap.c is deprecated and not recommended now. Use unimap instead.
